src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
+  height: 60px;
+  margin-right: 50px;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+li {
+  margin-right: 20px;
+}
+
+li:last-child {
+  margin-right: 0;
+}
+
+a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  text-align: left;
+  color: #000;
+  display: flex;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active {
```

Comment:
these styles should be applied to a link (not to helper class `is-active`). 
`is-active` should have only a color and be relative

Reply:
Not fixed